File: C/EX_estrutura_sequencial/ex2_retangulo/retangulo.c
/*
Exemplo 1: 
Base do retangulo: 4.0
Altura do retangulo: 5.0
AREA = 20.0000 
PERIMETRO = 18.0000 
DIAGONAL = 6.4031
*/


#include<stdio.h>
#include<math.h>
int main()
{
    double base, altura, area, perimetro, diagonal;

    printf("Base do retangulo: ");
    scanf("%lf", &base);

    printf("Altura do retangulo: ");
    scanf("%lf", &altura);

    area = base * altura;
    perimetro = (base + altura) * 2;
    diagonal = sqrt(base*base + altura*altura);

    printf("\nAREA = %.4lf", area);
    printf("\nPERIMETRO = %.4lf", perimetro);
    printf("\nDIAGONAL = %.4lf", diagonal);

    return 0;
}

File: Python/Ex estrutura sequencial/ex1 retangulo/retangulo.py
"""
Exemplo 1: 
Base do retangulo: 4.0
Altura do retangulo: 5.0
AREA = 20.0000 
PERIMETRO = 18.0000 
DIAGONAL = 6.4031 
"""
import math
base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = base * altura
perimetro = (base + altura) * 2
diagonal = math.sqrt(base ** 2 + altura ** 2)

print(f"AREA = {area:.4f}")
print(f"PERIMETRO = {perimetro:.4f}")
print(f"DIAGONAL = {diagonal:.4f}")